src/pymoca/parser.py

```diff
@@ -1033,12 +1035,13 @@ def parse(
 
         # Prune the database of entries not hit recently
         cursor.execute("BEGIN TRANSACTION;")
-        cutoff_time = int(
-            (datetime.utcnow() - timedelta(days=cache_expiration_days)).timestamp() * 1e6
-        )
+        cutoff_time = _microseconds_since_epoch(timedelta(days=-cache_expiration_days))
         cursor.execute("DELETE FROM models WHERE last_hit < ?", (cutoff_time,))
+        # Sometimes Windows time resolution is a bit coarse, so we make
+        # sure that if we update the last_prune time, it is actually newer
+        # than the previous one.
         cursor.execute(
-            "UPDATE metadata SET value = ? WHERE key = ?",
+            "UPDATE metadata SET value = max(value + 1, ?) + 1 WHERE key = ?",
```

Comment:
Comparing this to line 1080, there is an extra `+ 1` before the `WHERE`. Is that what you want?

Reply:
Ahh, that's a good catch! That +1 at the end is a remnant of me trying `? + 1` first, before realizing I could just do the proper thing with `max()`.